app/channels/application_cable/channel.rb

```diff
@@ -1,6 +0,0 @@
-# frozen_string_literal: true
```

Comment:
Not sure that it's a good idea to remove this code. This repo is a template for future projects, so it would be nice to have it.